ister_gui.py

```diff
@@ -644,6 +672,247 @@ def validate_condition(self):
         return True
 
 
+class ChooseAction(ProcessStep):
+    """UI to select installation path (Install, Shell, or Repair)"""
+    def __init__(self):
+        super(ChooseAction, self).__init__()
+        self.choices = {
+            'Install': 'Install          - Install Clear Linux OS',
+            'Repair': 'Repair           - Attempt to repair host OS',
+            'Shell': 'Shell            - Open shell in host OS',
+            'Update Installer': 'Update Installer - Update installer to '
```

Comment:
This option is unusual and I'm not super excited about bugs where the installer failed after an update to the installer image due to who knows what instead of just grabbing a new installer image and trying that instead. Could use some discussion on why we should add it.